docs/tutorial/fdfitting.rst

```diff
@@ -380,82 +440,59 @@ Note that this piece of code will lead to parameters `DNA/Lc_0`, `DNA/Lc_1` etc.
 Multiple models
 ***************
 
-When working with multiple models, things can get a little more complicated. Let's say we have two models, `model1` and
-`model2` and we want to fit both in a global fit. Constructing the :class:`~lumicks.pylake.FdFit` is easy::
+Sometimes, you need to fit multiple models, for example before and after unfolding of a protein.
+Let’s say we have two models, `model1` and `model2` and we want to fit both in a global fit.
+The first step is to construct the :class:`~lumicks.pylake.FdFit`::
 
     model1 = lk.ewlc_odijk_force("DNA")
     model2 = (lk.ewlc_odijk_distance("DNA") + lk.ewlc_odijk_distance("protein")).invert()
     fit = lk.FdFit(model1, model2)
 
-But then the question arises, how do we add data to each model? Well, the trick is in the assignments to `model1` and
-`model2`. We can use these now to add data to each model as follows::
+We add force-distance data from before the unfolding event from two different measurements:::
 
-    fit[model1].add_data("data for model 1", forces_1, distances_1)
-    fit[model2].add_data("data for model 2", forces_2, distances_2)
+    for i, (distance, force) in enumerate(zip(distances, forces)):
+        fit[model1].add_data(f"Before unfolding {i}", force, distance)
 
 See how we used the model handles? They are used to let the :class:`~lumicks.pylake.FdFit` know to which model each data set should be added.
-You can add as many data sets as you want to both models and fit it all at once.
-
-Plotting is straightforward in this setting. We can plot the data sets corresponding to model 1 and 2 as follows::
-
-    fit[model1].plot()
-    fit[model2].plot()
-
-Accessing the model parameters for a specific data set is a little more complicated in this setting. If we want to
-obtain the parameters for "data for model 1", we'd have to invoke::
+Rather then directly adding the data from after the unfolding event, and fitting everything together, we are going to fit in incrementally::
```

Comment:
```suggestion
Rather than directly adding the data from after the unfolding event, and fitting everything together, we are going to fit incrementally::
```